nats.go

```diff
@@ -3476,3 +3477,17 @@ func (nc *Conn) Barrier(f func()) error {
 	nc.mu.Unlock()
 	return nil
 }
+
+// GetCID returns the client ID assigned by the server the client
+// is currently connected to. Note that the value would change if
+// client reconnects. Also, returned value may be 0 if connecting
```

Comment:
Should return an error if not provided.

Reply:
Done, please comment on error variable name and content